src/components/_classes/component/Component.js

```diff
@@ -7,12 +7,13 @@ import isMobile from 'ismobilejs';
 import Formio from '../../../Formio';
 import * as FormioUtils from '../../../utils/utils';
 import Validator from '../../../validator/Validator';
+import TableModule from 'quill-table';
```

Comment:
Why can we not lazy load this?....

https://github.com/MrMadClown/quill-table/blob/master/dist/quill-table.js

We can add this to our CDN so that it is lazy-loadable, and the following should work.

```js
return Formio.requireLibrary('quill', 'Quill', _.get(this.options, 'editors.quill.src', `${QUILL_URL}/quill.min.js`), true)
      .then(() => {
return Formio.requireLibrary('quill-table', 'TableModule', `${QUILL_TABLE_URL}/quill.table.js`, true)
      .then(() => {

});
});
```